auth/user_mgt.go

```diff
@@ -475,6 +688,59 @@ func (c *Client) SetCustomUserClaims(ctx context.Context, uid string, customClai
 	return c.updateUser(ctx, uid, (&UserToUpdate{}).CustomClaims(customClaims))
 }
 
+// ImportUsers imports an array of users to Firebase Auth.
+//
+// No more than 1000 users can be imported in a single call. If at least one user specifies a
+// password, a UserImportHash must be specified as an option.
+func (c *Client) ImportUsers(ctx context.Context, users []*UserToImport, opts ...UserImportOption) (*UserImportResult, error) {
+	if len(users) == 0 {
+		return nil, fmt.Errorf("users list must not be empty")
```

Comment:
I'd recommend using `errors.New()` instead of `fmt.Errorf()` when you don't need any interpolation.

Reply:
Done